pkg/proxy/util/conntrack.go

```diff
@@ -31,17 +31,21 @@ const noConnectionToDelete = "0 flow entries have been deleted"
 
 // DeleteServiceConnection uses the conntrack tool to delete the conntrack entries
 // for the UDP connections specified by the given service IPs
-func DeleteServiceConnections(execer exec.Interface, svcIPs []string) {
+func DeleteServiceConnections(execer exec.Interface, svcIPs []string) error {
+	var errs []error
 	for _, ip := range svcIPs {
```

Comment:
Truthfully, this should probably be looping in the caller and this function should just run a single conntrack, so the caller can handle multiple errors cleanly.